src/kernel/include/pgtbl.h

```diff
@@ -10,7 +10,7 @@
 
 #include "shared/cos_errno.h"
 #include "ertrie.h"
-#include "shared/util.h"
+#include "chal/util.h"
```

Comment:
Need to abstract this file somewhere into platform code...not replace x86.  You're just making your life difficult for when you merge with mainline.